README.md

```diff
@@ -1,88 +1,102 @@
->>> servers = api.servers()
->>> servers[0].cve_announcements_count
-153
-```
+A simple interface for your Cyberwatch instance API.
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**  *generated with [DocToc](https://github.com/thlorenz/doctoc)*
+
+- [Cyberwatch API toolbox](#cyberwatch-api-toolbox)
+  - [Installation](#installation)
+    - [Prerequisites](#prerequisites)
+    - [Install the package](#install-the-package)
+    - [Test your installation](#test-your-installation)
+  - [Configuration](#configuration)
+  - [Usage](#usage)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Installation
 
-Installation
-------------
+### Prerequisites
+- [ ] [Python 3](https://www.python.org/)
+- [ ] Python [PIP](https://pypi.org/project/pip/)
 
-**Prerequisite**
-- [Python 3](https://www.python.org/)
-- Python [PIP](https://pypi.org/project/pip/)
+### Install the package
 
 To install Cyberwatch API toolbox, simply use python 3 with:
 
-``` {.sourceCode .bash}
+```bash
 $ pip3 install git+https://github.com/Cyberwatch/cyberwatch_api_toolbox
 ```
 
-Test your installation
-------------
-- Install the package (see above)
-- Configure the test script : 
-    - Create a new file called ```ping.py```
-    - Copy and paste : 
-    ``` {.sourceCode .python}
-    from cbw_api_toolbox.cbw_api import CBWApi
-    
-    API_KEY = ''
-    SECRET_KEY = ''
-    API_URL = ''
-    
-    CBWApi(API_URL, API_KEY, SECRET_KEY).ping()
-    ```
-    - Edit this script with **Configure** below.
-    - Launch the script with `python3 ping.py`
-    - You should see `OK`, if you have a `Failed` you should review your configuration.
-
-Configure
-------------
-- Enter your cyberwatch user api key in ```API_KEY```
-    
-*Example: API_KEY = 'PyXpxrcJ7rQ...'*
-
-- Enter your cyberwatch user api secret in ```SECRET_KEY```
-
-*Example: SECRET_KEY = '+bUx37WnB0qt...'*
-
-- Enter your cyberwatch instance url in ```API_URL```
-
-*Example: API_URL = 'https://myinstance.local'*
-
-- To find your api key and api secret key:
-    - Click on your profile page in your cyberwatch instance web page
-    - Click on the button 'see my api keys'
-
-Usage
-------------
-
-- To launch a script example:
-    - Install the package (see above)
-    - And then choose and copy an example to your computer from the [examples directory](examples)
-
-- To use this example you will need to **edit** the script with your configuration (see **Configure**)     
-
-- After this steps your file should look like : 
-``` {.sourceCode .python}
+### Test your installation
+
+**Create a new file called `ping.py` and copy/paste this content**
+
+```python
+# ping.py
 from cbw_api_toolbox.cbw_api import CBWApi
 
+API_KEY = ''
+SECRET_KEY = ''
+API_URL = ''
+
+CBWApi(API_URL, API_KEY, SECRET_KEY).ping()
+```
+
+**Configure the file with your credentials**
```

Comment:
Maybe 'environment' instead of 'credentials' because it also test the connection to the CBW instance